evergreen/Dockerfile

```diff
@@ -6,14 +6,22 @@ ENV PYTHONUNBUFFERED 1
 WORKDIR /root
 COPY pyproject.toml .
 RUN pip3 install poetry 
+
 RUN poetry config virtualenvs.create false --local
+
+# RUN apt-get update && apt-get install -y nginx
+# RUN rm /etc/nginx/conf.d/default.conf
+
```

Comment:
There's a bit of commented-out stuff like this that should just be deleted. You can look at all the changes on GitHub to see them all